src/scenes/City.ts

```diff
@@ -0,0 +1,28 @@
+import { ICity } from "./ICity";
+export class City implements ICity {
+    // economy= [stock, consume, production]
+    constructor(private name: string, private economy: number[]) {
```

Comment:
instead of number arrays, why not simply add three properties 

```typescript
private stock: number;
private consumption: number;
private production: number;
```

This improves readilibity of `getStock()` etc  greatly